string action_name
int32 action_code
float32 priority
int32 danger_sector
int32 danger_index
int8 real_target
int8 fake_target
int8 type
